Example/ExampleApp/SessionDetails/SessionDetailViewController.swift

```diff
@@ -25,21 +25,43 @@ final class SessionDetailViewController: UIHostingController<SessionDetailView>
         viewController.onSign = { [unowned self] in
             let result = Signer.signEth(request: request)
             let response = JSONRPCResponse<AnyCodable>(id: request.id, result: result)
-            Sign.instance.respond(topic: request.topic, response: .response(response))
+            respondOnSign(request: request, response: response)
             reload()
         }
         viewController.onReject = { [unowned self] in
-            Sign.instance.respond(
-                topic: request.topic,
-                response: .error(JSONRPCErrorResponse(
-                    id: request.id,
-                    error: JSONRPCErrorResponse.Error(code: 0, message: ""))
-                )
-            )
+            respondOnReject(request: request)
             reload()
         }
         present(viewController, animated: true)
     }
+    
+    private func respondOnSign(request: Request, response: JSONRPCResponse<AnyCodable>) {
+        print("[CONTROLLER] Respond on Sign")
```

Comment:
We could already replace "[controller]" in logs with "[wallet]" wherever we find it. The controller terminology is a lot more implicit and internal to the SDK, now. I think just using "wallet" and "dapp" will make easier for external devs to understand the code.